packages/askui-nodejs/src/utils/http/credentials.ts

```diff
@@ -1,6 +1,5 @@
 export interface CredentialArgs {
   workspaceId: string,
```

Comment:
Looks like we don't even need the `workspaceId` in here. Can we remove it? Also from construction?

Reply:
you mean inside class Credentials? 
the problem is, we need it only when the token Is defined and it's a part of the Credential. 
maybe we should encode it inside the token string, then it might be removed.

Reply:
you are right, I mean inside the class where we now can simply pass the token instead of credentials to the constructor.